recharge/order/add_free_product_to_upcoming_order/mesa.json

```diff
@@ -19,11 +19,12 @@
                 "trigger_type": "input",
                 "type": "recharge",
                 "entity": "charge",
-                "action": "charge/created",
+                "action": "charge\/created",
```

Comment:
@amandahom QA'd and works great! 🥳

Is this backward slash needed?

Also, your last three checkboxes aren't displaying properly in the QA checklist, but they should be checked.